packages/fast-colors/README.md

```diff
@@ -0,0 +1,33 @@
+# FAST colors
+A color utility library designed to make working with colors easier.
+
+## range(color, [options])
+A function that converts a single color input into a range of outputs, from lightest to darkest. 
+- `color: string` - a HEX or RGB color represented as a string (eg, `"#FFF"` or `"rgb(0, 0, 0)"`)
+- `options: object`
+    - `count: number` - the number of colors to create in the count. Defaults to `7`.
+    - `paddingLight: number` - the amount to reduce the color-range of light values by. Defaults to `0.185`.
+    - `paddingDark: number` - the amount to reduce the color-range of dark values by. Defaults to `0.185`.
+    - `saturationLight: number` - the amount to adjust the saturation of light values by. Defaults to `0.35`.
+    - `saturationDark: number` - the amount to adjust the saturation of light values by. Defaults to `1.25`.

```

Comment:
Looks like the `light` in this sentence should be `dark` instead.

Reply:
Fixed